continuous_integration/show-python-packages-versions.py

```diff
@@ -1,16 +1,16 @@
 import sys
+import pkg_resources
 
 DEPENDENCIES = ['numpy', 'scipy', 'scikit-learn', 'joblib', 'matplotlib',
                 'nibabel']
 
 
 def print_package_version(package_name, indent='  '):
     try:
-        package = __import__(package_name)
-        version = getattr(package, '__version__', None)
-        package_file = getattr(package, '__file__', )
-        provenance_info = '{0} from {1}'.format(version, package_file)
-    except ImportError:
+        dist = pkg_resources.get_distribution(package_name)
+        provenance_info = '{0} installed in {1}'.format(dist.version,
```

Comment:
This should be in the `else` block. `try` should only have the statement that may generate the exception we're looking for.